hikey960.mk

```diff
@@ -58,10 +58,12 @@ STRACE_PATH			?=$(ROOT)/strace
 # Targets
 ################################################################################
 .PHONY: all
-all: arm-tf boot-img lloader strace
+all: arm-tf boot-img lloader strace optee-examples
```

Comment:
minor: not necessary to have the target here since `common.mk` already takes care of it in `filelist-tee-common`.

Reply:
fixed